cfme/services/catalogs/__init__.py

```diff
@@ -53,8 +53,8 @@ class toolbar(View):  # noqa
         policy = Dropdown('Policy')
         view_selector = View.nested(ItemsToolBarViewSelector)
 
-    # for backward compatibility. it is difficult to figure out where those are used
-    # TODO: this should be fixed by this code owner
+    # TODO(nansari/sshveta): this should be fixed by this code owner
```

Comment:
Change `fixed by this code owner` to `fixed by the indicated code owner`

Reply:
I think better i remove this like `identifier` itself indicating owners.